.github/workflows/scripts/coverage.sh

```diff
@@ -0,0 +1,44 @@
+!#/usr/bin/env bash
+
+# Ensure past coverage files are deleted (useful for running locally)
+echo "Removing stale coverage files..."
+find . -name "*.profraw" -type f -delete
+find . -name "*.profdata" -type f -delete
+rm -r coverage
+mkdir coverage
```

Comment:
Can we actually remove them when the script is done running ? Otherwise all these files are remaining after a run

Reply:
It's alright, we can gitignore this one. But at the moment the script is leaving lots of files in all crates, a bit annoying.

Reply:
Maybe you can add coverage folder to gitignore then

Reply:
Agreed, I'll address the intermediate files and add to the gitignore 👍